.gitignore

```diff
@@ -4,3 +4,4 @@ public/lib
 .DS_store
 c4h_keys.txt
 /data
+/.env
```

Comment:
Please add a new line to the end of this file.

Reply:
It's the URL to the staging and testing database. It's stored in the .env file. Heroku auth generated that app.json file and commit it to the staging branch. Will remove before I raise any PR.